package.json

```diff
@@ -1,5 +1,6 @@
 {
     "dependencies": {
+        "chai": "^4.2.0",
```

Comment:
Is this irrelevant?

Reply:
The test requires chai. Should I remove it there?